llvm/include/llvm/IR/ModuleSummaryIndex.h

```diff
@@ -423,6 +427,16 @@ inline raw_ostream &operator<<(raw_ostream &OS, const AllocInfo &AE) {
   for (auto &M : AE.MIBs) {
     OS << "\t\t" << M << "\n";
   }
+  if (!AE.TotalSizes.empty()) {
+    OS << " TotalSizes per MIB:\n\t\t";
+    First = true;
+    for (auto &TS : AE.TotalSizes) {
```

Comment:
nit: There's no benefit to using a reference here since we don't mutate the value and the uint64 is just as wide as the pointer (on 64-bit platforms). Using auto hides this. Maybe spell out the type and drop the reference?

Reply:
done